unit-tests/live/safety/test-safety-metadata.py

```diff
@@ -2,28 +2,67 @@
 # Copyright(c) 2023 Intel Corporation. All Rights Reserved.
 
 #test:donotrun
-#test:device D585S
+# test:device D585S
 
 import pyrealsense2 as rs
 from rspy import test
 
+
+print("remi")
+
 safety_metadata_values = [rs.frame_metadata_value.frame_counter,
                           rs.frame_metadata_value.frame_timestamp,
                           rs.frame_metadata_value.safety_level1,
                           rs.frame_metadata_value.safety_level2]
 
+occupancy_metadata_values = [rs.frame_metadata_value.frame_counter,
+                             rs.frame_metadata_value.safety_depth_frame_counter,
+                             rs.frame_metadata_value.frame_timestamp,
+                             rs.frame_metadata_value.floor_plane_equation_a,
+                             rs.frame_metadata_value.floor_plane_equation_b,
+                             rs.frame_metadata_value.floor_plane_equation_c,
+                             rs.frame_metadata_value.floor_plane_equation_d,
+                             rs.frame_metadata_value.safety_preset_id,
+                             rs.frame_metadata_value.occupancy_grid_rows,
+                             rs.frame_metadata_value.occupancy_grid_columns,
+                             rs.frame_metadata_value.occupancy_cell_size]
+
+point_cloud_metadata_values = [rs.frame_metadata_value.frame_counter,
+                               rs.frame_metadata_value.safety_depth_frame_counter,
+                               rs.frame_metadata_value.frame_timestamp,
+                               rs.frame_metadata_value.floor_plane_equation_a,
+                               rs.frame_metadata_value.floor_plane_equation_b,
+                               rs.frame_metadata_value.floor_plane_equation_c,
+                               rs.frame_metadata_value.floor_plane_equation_d,
+                               rs.frame_metadata_value.safety_preset_id,
+                               rs.frame_metadata_value.number_of_3d_vertices]
+
+
 def check_md_value(frame, md_value):
     test.check(frame.supports_frame_metadata(md_value))
     val = frame.get_frame_metadata(md_value)
     rs.log(rs.log_severity.info, repr(md_value) + ": " + repr(val))
```

Comment:
What is this? librealsense log?
Why not test log?
`from rspy import test,log`
`log.i(...)`

Reply:
done

Reply:
Did you forgot to push the changes?

Reply:
seems so

Reply:
Sort of - now the changes are there.